### PR TITLE
DAT- 2544, send poi article link to native layer on click when map embeded in webvi...

### PR DESCRIPTION
**Pull request by RafLeszczynski**
...ew of wikia apps

### CODE REVIEW

File: assets/scripts/im.pontoWikiaAppsBrigde.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+			host = config.mapConfig.city_url.replace(/^http\:\/\//, ''),
+			i = links.length,
+			link,
+			path;
+
+		while(i--) {
+			link = links[i];
+			path = link.pathname || link.href;
+
+			if ((link.host && link.host !== host) || path === '/wikia.php') {
```
Comment (rogatty) [inline assets/scripts/im.pontoWikiaAppsBrigde.js]: Why `/wikia.php`? Can you add some comment what is happening here?

File: assets/scripts/im.pontoWikiaAppsBrigde.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+		});
+	}
+
+
+	/**
+	 * @desc setup POI article links for native wikia app
+	 */
+	function setupPoiLinks() {
+		var links = w.document.getElementsByClassName(poiArticleLinkClass),
+			host = config.mapConfig.city_url.replace(/^http\:\/\//, ''),
```
Comment (rogatty) [inline assets/scripts/im.pontoWikiaAppsBrigde.js]: I know we don't use https yet but let's be prepared for it.

File: assets/scripts/im.pontoWikiaAppsBrigde.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+			i = links.length,
+			link,
+			path;
+
+		while (i--) {
+			link = links[i];
+			path = link.pathname || link.href;
+
+			if ((link.host && link.host !== host) || path === '/wikia.php') {
```
Comment (rogatty) [inline assets/scripts/im.pontoWikiaAppsBrigde.js]: Why `/wikia.php`? Can you add some comment what is happening here?

File: assets/scripts/im.pontoWikiaAppsBridge.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+		poiArticleLinkClass = 'poi-article-link';
+
+	/**
+	 * @desc sends link URL to native wikia app
+	 * @param {Element} link
+	 */
+	function sendLinkUrl(link) {
+		ponto.invoke(pontoWikiaAppClass, 'goTo', {
+			url: link.href,
+			title: link.dataset.articleTitle.replace(/ /g, '_')
```
Comment (rogatty) [inline assets/scripts/im.pontoWikiaAppsBridge.js]: I would add some check here. We don't want to throw exceptions when there is no `link.dataset.articleTitle`

File: lib/renderMap.js [hunk 5]
```diff
@@ -335,6 +335,7 @@ function renderMap(req, res, next, apiPathEntryPoint, apiConfigUrl) {
 						mapData.iframeSrc = utils.responseUrl(req, apiPathEntryPoint + 'render/', mapId);
 						mapData.hideAttr = typeof(req.query.hideAttr) !== 'undefined';
 						mapData.cacheBuster = config.getCachebusterUrlPart();
+						mapData.isEmbeddedInWikiaApp = typeof(req.query.isEmbeddedInWikiaApp) !== 'undefined';
```
Comment (rogatty) [inline lib/renderMap.js]: So `&isEmbeddedInWikiaApp=false` will set `isEmbeddedInWikiaApp = true`? :)
I can see that `hideAttr` works the same way but I don't like it anyway. Your call.

Comment (RafLeszczynski) [inline lib/renderMap.js]: Discussed offline

**Comment by RafLeszczynski** [comment]
@rogatty could you please take a look one again?

**Comment by rogatty** [comment]
LGTM

**Pull request merged**